src/PatientCorrections-1-0/Cert/_reference/resources-generic/nl-core-Patient-1285290.xml

```diff
@@ -3,6 +3,14 @@
     <meta>
         <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient"/>
     </meta>
+   <identifier>
+        <system value="http://fhir.nl/fhir/NamingSystem/bsn" />
+        <value>
+            <extension url="http://hl7.org/fhir/StructureDefinition/data-absent-reason">
+                <valueCode value="masked" />
+            </extension>
+        </value>
+    </identifier>
     <identifier>
```

Comment:
Shouldn't this additional identifier be removed? (I wonder how this will affect the name of the file).

Reply:
Ik dacht dat dit in het ticket stond, maar dit is er bij in geschoten. Dus bij deze:

Dossierwijzigingsverzoek is de enige standaard waarbij de usecase van deze extra identifier duidelijk staat toegelicht op de wiki: https://informatiestandaarden.nictiz.nl/wiki/MedMij:V6/KwalificatiescriptDossierwijzigingsverzoekSturen#Persoonsgegevens_2
Dus dat rechtvaardigt het wat mij betreft om deze (als enige) te laten staan in de FHIR-materialen. Misschien is dat alsnog het rechttrekken (verwijderen) waard, maar dat leek mij niet aan mij binnen dit ticket, want dan moeten FHIR en wiki tegelijk wat ingrijpender worden aangepast.

Reply:
Helder!